### DISCUSSION TITLE
Open Start on monitor which cursor is on issues

### DISCUSSION THREAD

**Discussion by JTorreG** [valinet/ExplorerPatcher#1864] [General]
When enabling the option: _Start menu > When using multiple displays [...]: Monitor on which the cursor is on_ , the start menu on the secondary monitor is not properly displayed.

Main screen (5120x1440)
![image](https://github.com/valinet/ExplorerPatcher/assets/78599298/2c92bd30-d2be-47a6-91ad-2091a83c40f4)

Secondary Screen (1920x1080)
![image](https://github.com/valinet/ExplorerPatcher/assets/78599298/3bc6c2ce-e2f1-4945-a9ab-60b8fd1749d6)

The Start Menu in the secondary screen is rendering overlapping the task bar and goes outside the screen.

**Comment by pyrates999** [comment]
Is this the same thing? https://github.com/valinet/ExplorerPatcher/issues/1759
